--- gridengine/source/3rdparty/sge_depend/def.h.getline	2003-03-08 22:06:35.000000000 -0700
+++ gridengine/source/3rdparty/sge_depend/def.h	2009-04-06 13:06:37.000000000 -0600
@@ -136,7 +136,7 @@
 #if NeedFunctionPrototypes
 char *copy( char *str);
 char *base_name(char *file, int strip);
-char *getline(struct filepointer *filep);
+char *sge_getline(struct filepointer *filep);
 struct filepointer *getfile(char *file);
 struct inclist *newinclude(char *newfile, char *incstring);
 struct inclist *inc_path(char	*file, char *include, boolean	dot);
@@ -155,7 +155,7 @@
 #else
 char *copy();
 char *base_name();
-char *getline();
+char *sge_getline();
 struct filepointer *getfile();
 struct inclist *newinclude();
 struct inclist *inc_path();
--- gridengine/source/3rdparty/sge_depend/parse.c.getline	2008-04-16 03:11:05.000000000 -0600
+++ gridengine/source/3rdparty/sge_depend/parse.c	2009-04-06 13:06:45.000000000 -0600
@@ -80,7 +80,7 @@
    register char   *line;
    register int   type;
 
-   while ((line = getline(filep))) {
+   while ((line = sge_getline(filep))) {
       switch(type = deftype(line, filep, file_red, file, FALSE)) {
       case IF:
       case IFFALSE:
@@ -571,7 +571,7 @@
    register int   type;
    boolean recfailOK;
 
-   while ((line = getline(filep))) {
+   while ((line = sge_getline(filep))) {
       switch(type = deftype(line, filep, file_red, file, TRUE)) {
       case IF:
       doif:
--- gridengine/source/3rdparty/sge_depend/main.c.getline	2007-07-26 09:41:43.000000000 -0600
+++ gridengine/source/3rdparty/sge_depend/main.c	2009-04-06 13:06:49.000000000 -0600
@@ -510,7 +510,7 @@
  * Get the next line.  We only return lines beginning with '#' since that
  * is all this program is ever interested in.
  */
-char *getline(filep)
+char *sge_getline(filep)
    register struct filepointer   *filep;
 {
    register char   *p,   /* walking pointer */

